src/ROUTES.ts

```diff
@@ -262,10 +262,6 @@ const ROUTES = {
         route: ':iouType/new/participants/:reportID?',
         getRoute: (iouType: string, reportID = '') => `${iouType}/new/participants/${reportID}` as const,
     },
-    MONEY_REQUEST_CONFIRMATION: {
-        route: ':iouType/new/confirmation/:reportID?',
-        getRoute: (iouType: string, reportID = '') => `${iouType}/new/confirmation/${reportID}` as const,
-    },
     MONEY_REQUEST_DATE: {
```

Comment:
This causes a regression. The Send money flow still uses this route.


https://github.com/Expensify/App/assets/16493223/1141e934-1f29-455d-8814-d854983cdb26

Reply:
This crash is only temporary because we haven't migrated the rest of this refactor cleanup yet. I will create a new PR to temporarily fix those places to use the new route